src/NUnitFramework/Directory.Build.props

```diff
@@ -17,11 +18,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <DefineConstants Condition="'$(TargetFramework)' != 'netstandard2.0'
-                            and '$(TargetFramework)' != 'netcoreapp2.1'
-                            and '$(TargetFramework)' != 'netcoreapp3.1'
-                            and '$(TargetFramework)' != 'net5.0'
-                            and '$(TargetFramework)' != 'net5.0-windows'">$(DefineConstants);THREAD_ABORT</DefineConstants>
+    <DefineConstants Condition="$(TargetFramework.StartsWith('net4'))">$(DefineConstants);THREAD_ABORT</DefineConstants>
```

Comment:
Option: `Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'"`

The parallel of `#if NETFRAMEWORK`

Reply:
Thanks, code updated.